ush/nems_configure.sh

```diff
@@ -7,123 +7,123 @@
-sed -i -e "s;@\[med_petlist_bounds\];$med_petlist_bounds;g" tmp1
-sed -i -e "s;@\[atm_petlist_bounds\];$atm_petlist_bounds;g" tmp1
-sed -i -e "s;@\[esmf_logkind\];$esmf_logkind;g" tmp1
+sed -i -e "s;@\[atm_model\];fv3;g" tmp1
+sed -i -e "s;@\[med_petlist_bounds\];${med_petlist_bounds};g" tmp1
+sed -i -e "s;@\[atm_petlist_bounds\];${atm_petlist_bounds};g" tmp1
+sed -i -e "s;@\[esmf_logkind\];${esmf_logkind};g" tmp1
 
-if [ $cpl = ".true." ]; then
-  sed -i -e "s;@\[coupling_interval_slow_sec\];$CPL_SLOW;g" tmp1
+if [[ ${cpl} = ".true." ]]; then
+  sed -i -e "s;@\[coupling_interval_slow_sec\];${CPL_SLOW};g" tmp1
 fi
 
-if [ $cplflx = .true. ]; then
-  if [ $restart_interval  -gt 0 ]; then
-    restart_interval_nems=$restart_interval
+if [[ ${cplflx} = .true. ]]; then
+  if [[ ${res_int}  -gt 0 ]]; then
+    local restart_interval_nems=${res_int}
   else
-    restart_interval_nems=$FHMAX
+    local restart_interval_nems=${FHMAX}
   fi
-  sed -i -e "s;@\[ocn_model\];$OCN_model;g" tmp1
-  sed -i -e "s;@\[ocn_petlist_bounds\];$ocn_petlist_bounds;g" tmp1
-  sed -i -e "s;@\[DumpFields\];$DumpFields;g" tmp1
-  sed -i -e "s;@\[cap_dbug_flag\];$cap_dbug_flag;g" tmp1
-  sed -i -e "s;@\[use_coldstart\];$use_coldstart;g" tmp1
-  sed -i -e "s;@\[RUNTYPE\];$cmeps_run_type;g" tmp1
-  sed -i -e "s;@\[CPLMODE\];$cplmode;g" tmp1
-  sed -i -e "s;@\[restart_interval\];$restart_interval;g" tmp1
-  sed -i -e "s;@\[coupling_interval_fast_sec\];$CPL_FAST;g" tmp1
-  sed -i -e "s;@\[RESTART_N\];$restart_interval_nems;g" tmp1
-  sed -i -e "s;@\[use_mommesh\];$USE_MOMMESH;g" tmp1
-  sed -i -e "s;@\[eps_imesh\];$EPS_IMESH;g" tmp1
-  sed -i -e "s;@\[ATMTILESIZE\];$RESTILE;g" tmp1
+
+  case "${OCNRES}" in
+    "500") local eps_imesh="4.0e-1";;
+    "100") local eps_imesh="2.5e-1";;
+    *) local eps_imesh="1.0e-1";;
+  esac
+
+  local use_coldstart=${use_coldstart:-".false."}
+  local use_mommesh=${USE_MOMMESH:-"true"}
+  local restile=$(echo "${CASE}" |cut -c2-)
+  local ocn_petlist_bounds="${ATMPETS} $(( ${ATMPETS}+${OCNPETS}-1 ))"
+
+  sed -i -e "s;@\[ocn_model\];mom6;g" tmp1
+  sed -i -e "s;@\[ocn_petlist_bounds\];${ocn_petlist_bounds};g" tmp1
+  sed -i -e "s;@\[DumpFields\];${DumpFields};g" tmp1
+  sed -i -e "s;@\[cap_dbug_flag\];${cap_dbug_flag};g" tmp1
+  sed -i -e "s;@\[use_coldstart\];${use_coldstart};g" tmp1
+  sed -i -e "s;@\[RUNTYPE\];${cmeps_run_type};g" tmp1
+  sed -i -e "s;@\[CPLMODE\];${cplmode};g" tmp1
+  sed -i -e "s;@\[restart_interval\];${res_int};g" tmp1
+  sed -i -e "s;@\[coupling_interval_fast_sec\];${CPL_FAST};g" tmp1
+  sed -i -e "s;@\[RESTART_N\];${restart_interval_nems};g" tmp1
+  sed -i -e "s;@\[use_mommesh\];${use_mommesh};g" tmp1
+  sed -i -e "s;@\[eps_imesh\];${eps_imesh};g" tmp1
+  sed -i -e "s;@\[ATMTILESIZE\];${restile};g" tmp1
 fi
-if [ $cplwav = .true. ]; then
+
+if [[ ${cplwav} = .true. ]]; then
+
+  local wav_petlist_bounds="$(( ${ATMPETS}+${OCNPETS}+${ICEPETS} )) $(( ${ATMPETS}+${OCNPETS}+${ICEPETS}+${WAVPETS}-1 ))"
```

Comment:
OCNPETS is no longer defaulting to zero, so I think this fails if waves are on but ocean is not.

Reply:
done.